website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -102,9 +102,7 @@ properly follow the design goals set by [controller-runtime][controller-runtime]
 
 ### Define the API
 
-To begin, we will represent our API by defining the `Memcached` type, which will have a `MemcachedSpec.Size` field to set the quantity of memcached instances (CRs) to be deployed, and a `MemcachedStatus.Nodes` field to store a CR's Pod names.
-
-**Note** The Node field is just to illustrate an example of a Status field. In real cases, it would be recommended to use [Conditions][conditionals].
+To begin, we will represent our API by defining the `Memcached` type, which will have a `MemcachedSpec.Size` field to set the quantity of memcached instances (CRs) to be deployed, and a `MemcachedStatus.Nodes` field to store a CR's [Conditions][conditionals].
```

Comment:
Looking at the context in which this is being used, it appears to me that the `MemcachedStatus.Nodes` field is no longer being used but is now `MemcachedStatus.Conditions`:

```suggestion
To begin, we will represent our API by defining the `Memcached` type, which will have a `MemcachedSpec.Size` field to set the quantity of memcached instances (CRs) to be deployed, and a `MemcachedStatus.Conditions` field to store a CR's [Conditions][conditionals].
```

Reply:
My bad, let me fix this.

Reply:
Thanks @chiukapoor !